query/src/main/java/net/md_5/bungee/query/QueryHandler.java

```diff
@@ -106,18 +123,18 @@ protected void channelRead0(ChannelHandlerContext ctx, DatagramPacket msg) throw
                 } );
                 Map<String, String> data = new LinkedHashMap<>();
 
-                data.put( "hostname", listener.getMotd() );
+                data.put( "hostname", query.getMotd() );
                 data.put( "gametype", "SMP" );
                 // Start Extra Info
                 data.put( "game_id", "MINECRAFT" );
-                data.put( "version", bungee.getGameVersion() );
-                data.put( "plugins", "" );
+                data.put( "version", query.getVersion() );
+                data.put( "plugins", query.getPlugins().isEmpty() ? "" : query.getServer() + ": " + Joiner.on( "; " ).join( query.getPlugins() ) );
```

Comment:
Same concept. Some servers don't want their plugins visible.

Reply:
This list is empty if a plugin does not add anything